src/components/search/Search.js

```diff
@@ -29,6 +35,27 @@ const getViewportBounds = async (placeId) => {
     sw: { lat: sw.lat(), lng: sw.lng() },
   }
 }
+const CurrentLocationButton = (props) => (
+  <button {...props}>
+    <CurrentLocation size={24} />
+  </button>
+)
+const StyledCurrentLocationButton = styled(CurrentLocationButton)`
+  background: none;
+  color: inherit;
+  border: none;
+  height: 100%;
+  border-radius: 50% 0 0 50%;
+  border-right: 1px solid #e0e1e2;
+
+  svg {
+    color: ${({ theme }) => theme.blue};
+  }
+
+  &:disabled svg {
+    color: ${({ theme }) => theme.tertiaryText};
+  }
```

Comment:
Good work with the prepend and append!

However, you shouldn't have to style the border of the prepend. That should go in the Input. Think about it. If you were using the Input component, you shouldn't be expected to style the border-right, radius, etc. You should only have to think about styling the thing that you're putting inside the prepend or append of the Input.